core/iwasm/libraries/debug-engine/gdbserver.c

```diff
@@ -98,6 +98,14 @@ wasm_launch_gdbserver(char *host, int port)
         goto fail;
     }
 
+    socklen = sizeof(addr);
+    if (getsockname(listen_fd, (void *)&addr, &socklen) == -1) {
+        LOG_ERROR("%s", strerror(errno));
+        goto fail;
+    }
+
+    LOG_WARNING("Listening on %s:%d\n", host, ntohs(addr.sin_port));
```

Comment:
And shall we update the iwasm help and source debug document, to mention that when port is 0, a random port will be used?

Reply:
Done